src/devtools/client/debugger/src/components/PrimaryPanes/SourcesTreeItem.tsx

```diff
@@ -174,7 +174,7 @@ class SourceTreeItem extends Component<FinalSTIProps> {
   renderItemTooltip() {
     const { item, depth } = this.props;
 
-    return item.type === "source" ? unescape(item.contents.url) : getPathWithoutThread(item.path);
+    return item.type === "source" ? unescape(item.contents.url!) : getPathWithoutThread(item.path);
```

Comment:
I am pretty sure that console evals can be a source and not have a url

Reply:
I can tweak it, but this is the same logic that was there already. Only thing that changed is that `SourceTreeItem` now actually knows the right type for `TreeNode` and `SourceDetails`, instead of being `type $FixTypeLater = any`.

Reply:
come to think of it, we probably dont add sources without urls to the tree so it'll be caught higher up